website/docusaurus.config.js

```diff
@@ -6,8 +6,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula')
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'js-did',
-  tagline: 'Decentralized Identifiers tools',
+  title: 'Decentralized Identifiers (DIDs)',
```

Comment:
this also populates the top left title?

Reply:
It doesn't, @oed . Do you want me to put 'DIDs' in top left title? I'm thinking that 'Decentralized Identifiers (DIDs)' may be a bit too long there.

Reply:
Ok, not sure how it would look. Whatever you think is best.

Reply:
Put 'Decentralized Identifiers (DIDs)', we can always change it, if we feel it's too long there. Thanks, @oed :)!